pkg/network/telemetry/ebpf_telemetry.go

```diff
@@ -275,7 +308,7 @@ func (b *EBPFTelemetry) initializeMapErrTelemetryMap(maps []*manager.Map) error
 }
 
 func (b *EBPFTelemetry) initializeHelperErrTelemetryMap(probes []*manager.Probe) error {
-	z := new(HelperErrTelemetry)
+	var z []HelperErrTelemetry
```

Comment:
Why is this a slice now?

Reply:
Because the maps are per cpu now, and those require a slice.
https://github.com/cilium/ebpf/blob/main/marshalers.go#L165

Reply:
But you're passing it to the `Put` call below as is. Shouldn't it be something like `unsafe.Pointer(&z[0])`?

Reply:
We can pass a slice as is, for example:
 https://github.com/cilium/ebpf/blob/main/map_test.go#L1433-L1434